yew/src/services/dialog.rs

```diff
@@ -41,4 +44,45 @@ impl DialogService {
             feature = "web_sys" => utils::window().confirm_with_message(message).unwrap(),
         }
     }
+
+    /// Prompts the user to input a message. In most browsers this will open an alert box with
+    /// an input field where the user can input a message.
+    ///
+    /// It is possible to supply a default value which will be used if the user does not provide
+    /// a value.
+    ///
+    /// [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
+    ///
+    /// This method will `panic!` if there is an error in the process of trying to carry out this
+    /// operation.
+    ///
+    /// Note that this function is blocking; no other code can be run on the thread while
+    /// the user inputs their message which means that the page will appear to have 'frozen'
+    /// while the user types in their message.
+    pub fn prompt(
+        message: &str,
+        #[cfg(feature = "web_sys")] default: Option<&str>,
+        #[cfg(feature = "std_web")] _: Option<&str>,
```

Comment:
Do we need this argument on `std_web`?

Reply:
We don't, but it gets messy if functions have different signatures IMO.

Reply:
We generate separate docs for `stdweb` so I think it should be ok, do you have other concerns?

Reply:
Nope. I'll remove the argument on `stdweb` (+ add a note in the docstring).

Reply:
Oh, I didn't think about the doc string! You can check out `src/lib.rs` for example on how to render conditional docs